src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatEntryComponents = props.entries.map((entry, index) => {
+        return (
+            <li key={index}>
+                <ChatEntry 
```

Comment:
Note that it's best practice not to use the index of an item as the list item key in React since the order of items in an array can change. I'd recommend using the entry ID as the key. Also, to cut down on the number of tags, you can put the key attribute directly on the `<ChatEntry>` component and drop the `<li>` tag. This means you also won't need the `<ul>` tag on line 24.